components/ui/RadioButton.tsx

```diff
@@ -7,13 +7,15 @@ interface RadioButtonProps {
   inputGroup: string;
   inputClassName?: string;
   labelClassName?: string;
+  register: any;
```

Comment:
don't use `any`, ever. You can look into the type of the register function from the form lib. Ping me if you need more directions on how to do this, it's not exactly easy at first